File: pyramid.c
#include<stdio.h>

int main()
{
    int rows;
    printf("Enter the number of rows: ");
    scanf("%i",&rows);

    for (int i=1; i<=rows; i++)
    {
        for (int j=rows-1; j>=i; j--)
        {
            printf(" ");
        }
        for (int k=2*i-1; k>=1; k--)
        {
            printf("*");
        }
        printf("\n");
    }
    return 0;
}

File: pyramid.py
rows=int(input("Enter the number of rows: "))

for i in range (1,rows+1):
    for j in range (1,rows):
        if j>=i:
            print(" ",end='')
    for k in range (1,2*i):
        print("*",end='')
    print("\n")
